shlr/windbg/kd.h

```diff
@@ -203,6 +204,31 @@ typedef struct kd_packet_t {
 	uint8_t data[0];
 }) kd_packet_t;
 
+// KDNET
+
+#define KDNET_MAGIC 0x4d444247 // MDBG
+#define KDNET_DATA_SIZE 8
+#define KDNET_HMAC_SIZE 16
+
+// kdnet packet format
+// kdnet_packet_t
+// kdnet_data - encrypted
+// kd_packet - encrypted
+// kdnet_hmac
```

Comment:
A high level review would help the reader understand what's going on here. Or at least a link to some documentation on these fields and what's supposed to be in them.